main/options/options.cc

```diff
@@ -627,6 +629,9 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                  "Packages which are allowed to ignore the restrictions set by `visible_to` "
                                  "and `export` directives",
                                  cxxopts::value<vector<string>>(), "<name>");
+    options.add_options(section)("packager-layer", "Valid layer names for packages, ordered lowest to highest.",
+                                 cxxopts::value<vector<string>>()->default_value("library,application"),
+                                 "<layer-name>");
```

Comment:
Do we expect it to be more common for people to pass layers by repeatedly calling this option, or by passing a `,`-separated list?

I'm expecting that people will opt for the `,`-separated list, and thus a name of `packager-layers` (plural) will read better.

What do you think? Another option would be to think of a name that's somewhat agnostic of whether it's plural or not, but I can't think of any.

I'd like to spend the time to think about this now so we don't regret a name in the future and have to change it.

Reply:
I agree, I think the `,` separated list will probably be more common, so plural sounds better.

Do you have any thoughts on this part?

> it doesn't really convey that these are the possible layers, and not the layer being assigned to a package. Maybe --packager-possible-layers or --packager-valid-layers is better?